theano/gof/tests/test_vm.py

```diff
@@ -331,6 +331,9 @@ def time_linker(name, linker):
 
 
 class RunOnce(theano.Op):
+
+    __props__ = ()
+    
```

Comment:
It does not really make a difference here, but I would still include `'nb_run'` in `__props__`.